kyo-scheduler/jvm/src/main/scala/kyo/scheduler/Scheduler.scala

```diff
@@ -74,55 +68,54 @@ final class Scheduler(
         ExecutionContext.fromExecutor(asExecutor)
 
     private def schedule(task: Task, submitter: Worker): Unit = {
-        val cycles = this.cycles
-        @tailrec def loop(tries: Int = 0): Unit = {
```

Comment:
This method used to loop only when the current worker wasn't available (missing check below) or the candidate became unavailable between its election and the enqueuing, which isn't a relevant edge case.